.editorconfig

```diff
@@ -1,5 +1,5 @@
 [*.java]
 charset = utf-8
 indent_style = space
-indent_size = 2
+indent_size = 4
```

Comment:
Please don't change or commit the editor config file.

Reply:
my bad, fixed it